etc/swtpm-localca.conf

```diff
@@ -0,0 +1,3 @@
+statedir=${AUTOHCK_STATEDIR}
+signingkey=${AUTOHCK_STATEDIR}/signkey.pem
+issuercert = ${AUTOHCK_STATEDIR}/issuercert.pem
```

Comment:
Do we need certserial or this is optional?

Reply:
It is optional.